src/test/java/api/support/fixtures/TenantAPIFixture.java

```diff
@@ -0,0 +1,31 @@
+package api.support.fixtures;
+
+import static api.support.APITestContext.circulationModuleUrl;
+
+import org.folio.circulation.support.http.client.Response;
+import org.folio.rest.tools.PomReader;
+
+import api.support.RestAssuredClient;
+import io.vertx.core.json.JsonObject;
+
+public class TenantAPIFixture {
```

Comment:
I think we changed the production language to use activation, could we do the same here?

```suggestion
public class TenantActivationFixture {
```

Reply:
Renamed